scripts/gdb-pretty-printers.py

```diff
@@ -0,0 +1,151 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import gdb
+
+
+if not 'ThrustVectorPrinter' in locals():
+    raise Exception("This file expects the Thrust pretty-printers to be loaded already. "
```

Comment:
Always try to use a specific exception class rather than `Exception`. I think [NameError](https://docs.python.org/3/library/exceptions.html#NameError) is most appropriate here. If you don't know what to pick, `RuntimeError` is always a good fallback choice instead of `Exception`.
```suggestion
    raise NameError("This file expects the Thrust pretty-printers to be loaded already. "
```

Reply:
Actually, since we don't even depend on any code from the Thrust pretty-printer yet, we could also remove this error. This only means that people who load this script directly are missing out on some of the printers.

Reply:
Well, then. It would make a lot of sense to remove the error. 😄